Compilation fails with gcc-10

```
/usr/bin/ld: notebook.o:/builddir/build/BUILD/ddccontrol-0.4.4/src/gddccontrol/notebook.h:46: multiple definition of `mon'; main.o:/builddir/build/BUILD/ddccontrol-0.4.4/src/gddccontrol/notebook.h:46: first defined here
/usr/bin/ld: notebook.o:/builddir/build/BUILD/ddccontrol-0.4.4/src/gddccontrol/notebook.h:48: multiple definition of `main_app_window'; main.o:/builddir/build/BUILD/ddccontrol-0.4.4/src/gddccontrol/notebook.h:48: first defined here
/usr/bin/ld: notebook.o:/builddir/build/BUILD/ddccontrol-0.4.4/src/gddccontrol/notebook.h:97: multiple definition of `refresh_controls_button'; main.o:/builddir/build/BUILD/ddccontrol-0.4.4/src/gddccontrol/notebook.h:97: first defined here
/usr/bin/ld: notebook.o:/builddir/build/BUILD/ddccontrol-0.4.4/src/gddccontrol/notebook.h:50: multiple definition of `monitor_manager'; main.o:/builddir/build/BUILD/ddccontrol-0.4.4/src/gddccontrol/notebook.h:50: first defined here
/usr/bin/ld: notebook.o:/builddir/build/BUILD/ddccontrol-0.4.4/src/gddccontrol/notebook.h:51: multiple definition of `profile_manager'; main.o:/builddir/build/BUILD/ddccontrol-0.4.4/src/gddccontrol/notebook.h:51: first defined here
/usr/bin/ld: notebook.o:/builddir/build/BUILD/ddccontrol-0.4.4/src/gddccontrol/notebook.h:101: multiple definition of `num_monitor'; main.o:/builddir/build/BUILD/ddccontrol-0.4.4/src/gddccontrol/notebook.h:101: first defined here
/usr/bin/ld: notebook.o:/builddir/build/BUILD/ddccontrol-0.4.4/src/gddccontrol/notebook.h:100: multiple definition of `current_monitor'; main.o:/builddir/build/BUILD/ddccontrol-0.4.4/src/gddccontrol/notebook.h:100: first defined here
/usr/bin/ld: notebook.o:/builddir/build/BUILD/ddccontrol-0.4.4/src/gddccontrol/notebook.h:96: multiple definition of `cancelprofile_button'; main.o:/builddir/build/BUILD/ddccontrol-0.4.4/src/gddccontrol/notebook.h:96: first defined here
...
```
This is caused by a change in GCC 10 defaults from -fcommon to -fno-common [1]:

GCC now defaults to -fno-common. As a result, global variable accesses are more efficient on various targets. In C, global variables with multiple tentative definitions now result in linker errors. With -fcommon such definitions are silently merged during linking. 

If compiled with -fcommon it builds OK. The better solution is to fix the code to define the variables just once.

[1] https://gcc.gnu.org/gcc-10/changes.html

--- a/src/gddccontrol/main.c
+++ b/src/gddccontrol/main.c
@@ -47,6 +47,21 @@ GtkWidget* choice_hbox = NULL;
 GtkWidget* profile_hbox = NULL;
 GtkWidget* bottom_hbox = NULL;
 
+GtkWidget* main_app_window = NULL;
+
+GtkWidget* monitor_manager = NULL;
+GtkWidget* profile_manager = NULL;
+
+GtkWidget* profile_manager_button = NULL;
+GtkWidget* saveprofile_button = NULL;
+GtkWidget* cancelprofile_button = NULL;
+GtkWidget* refresh_controls_button = NULL;
+
+struct monitor* mon;
+
+int current_monitor; /* current monitor */
+int num_monitor; /* total number of monitors */
+
 struct monitorlist* monlist;
 
 GtkTooltips *tooltips;

--- a/src/gddccontrol/notebook.h
+++ b/src/gddccontrol/notebook.h
@@ -43,12 +43,12 @@
 #endif
 
 /* globals */
-struct monitor* mon;
+extern struct monitor* mon;
 
-GtkWidget* main_app_window;
+extern GtkWidget* main_app_window;
 
-GtkWidget* monitor_manager;
-GtkWidget* profile_manager;
+extern GtkWidget* monitor_manager;
+extern GtkWidget* profile_manager;
 
 /* notebook.c */
 
@@ -91,13 +91,13 @@ void set_message_ok(char* message, int with_ok);
 
 GtkWidget *stock_label_button(const gchar * stockid, const gchar *label_text, const gchar *tool_tip);
 
-GtkWidget* profile_manager_button;
-GtkWidget* saveprofile_button;
-GtkWidget* cancelprofile_button;
-GtkWidget* refresh_controls_button;
+extern GtkWidget* profile_manager_button;
+extern GtkWidget* saveprofile_button;
+extern GtkWidget* cancelprofile_button;
+extern GtkWidget* refresh_controls_button;
 
 /* Multimonitor support */
-int current_monitor; /* current monitor */
-int num_monitor; /* total number of monitors */
+extern int current_monitor; /* current monitor */
+extern int num_monitor; /* total number of monitors */
 
 #endif //NOTEBOOK_H

